phpBB/includes/mcp/mcp_main.php

```diff
@@ -1569,7 +1569,8 @@ function mcp_fork_topic($topic_ids)
 					$sql = 'SELECT * FROM ' . ATTACHMENTS_TABLE . "
 						WHERE post_msg_id = {$row['post_id']}
 							AND topic_id = $topic_id
-							AND in_message = 0";
+							AND in_message = 0
+						ORDER BY attach_id ASC";
```

Comment:
May I suggest to rewrite this query as in

```
					$sql = 'SELECT *
						FROM ' . ATTACHMENTS_TABLE . '
						WHERE post_msg_id = ' . (int) $row['post_id'] . '
							AND topic_id = ' . (int) $topic_id . '
							AND in_message = 0
						ORDER BY attach_id ASC';
```

Reply:
Makes sense.

Reply:
I am also not fond of constants but hey... no probs. ;)